lib/stoplight/light/runnable.rb

```diff
@@ -52,18 +52,12 @@ def run_code(on_success, on_failure)
         failures = clear_failures
         on_success.call(failures) if on_success
         result
-      rescue *[StandardError].concat(blacklisted_errors) => error
+      rescue Exception => error # rubocop:disable Lint/RescueException
```

Comment:
In general I agree, but RuboCop ultimately offers suggestions. In this case, I explicitly want to catch all exceptions and then manually select some to re-raise.